CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Change
+
+- **Company Data**
+  - show error message in company data edit overlay screen [#1385] (https://github.com/eclipse-tractusx/portal-frontend/pull/1385)
+
```

Comment:
Please update changelog in PR description as per https://github.com/eclipse-tractusx/portal/wiki/Example-of-Changelog-entry-in-PR-description

Reply:
updated